services/service-core/src/auth/keycloak.validator.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Humanitech Supply Trail
+ *
+ * Copyright (c) Humanitech, Peter Rogov and Contributors
+ *
+ * Website: https://humanitech.net
+ * Repository: https://github.com/humanitech-net/supply-trail
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import Joi from 'joi';
+export const userInputValidator = Joi.object({
```

Comment:
Is it really necessary to have all fields as required? What about updating the user?

Reply:
I don't think this object will be used later.
i will fix this one ".or('firstName', 'lastName', 'username')"